src/main/scala/uk/gov/nationalarchives/aws/utils/StepFunctionUtils.scala

```diff
@@ -16,6 +16,15 @@ class StepFunctionUtils(client: SfnAsyncClient) {
 
     IO(client.sendTaskSuccess(request)).futureLift
   }
+
+  def sendTaskFailureRequest(taskToken: String, error: String): IO[SendTaskFailureResponse] = {
+    val request = SendTaskFailureRequest.builder
+      .taskToken(taskToken)
+      .error(error)
+      .build
+
+    IO(client.sendTaskFailure(request)).futureLift
```

Comment:
I realise your method is doing the same thing as the existing ones, but is this using IO in a way that actually takes advantage of IO's lazy evaluation? It looks like we run the Future immediately and then wrap it in an IO.

Actually, I think this applies to some of the changes I made in the export project a few weeks ago too... We might need to talk to Sam about this. I'll make a note to ask him next week.